runconfig/errors.go

```diff
@@ -31,6 +31,8 @@ const (
 	ErrUnsupportedNetworkAndAlias validationError = "network-scoped alias is supported only for containers in user defined networks"
 	// ErrConflictUTSHostname conflict between the hostname and the UTS mode
 	ErrConflictUTSHostname validationError = "conflicting options: hostname and the UTS mode"
+	// ErrEmptyConfig when container config is nil
+	ErrEmptyConfig validationError = "Config cannot be empty in order to create a container"
```

Comment:
At least not the CLI. Is that enough to consider fixing the capitalization?

Reply:
Yes, probably; if CI still passes on 17.06 (and `docker-py`), we're probably good to go.

I'm ok with doing it separately though; perhaps better for this case to have a clear commit where we changed it (in case we missed it being used somewhere, and need to find a regression point)